tdrs-backend/tdpservice/reports/views.py

```diff
@@ -18,24 +22,42 @@ class ReportFileViewSet(ModelViewSet):
 
     http_method_names = ['get', 'post', 'head']
     parser_classes = [MultiPartParser]
-    permission_classes = [CanUploadReport]
+    permission_classes = [ReportFilePermissions]
     serializer_class = ReportFileSerializer
     queryset = ReportFile.objects.all()
 
+    @action(methods=["get"], detail=True)
+    def download(self, request, pk=None):
```

Comment:
By making this a detail action we can leverage some DRF built in capabilities, such as `get_object` to retrieve the report being requested and automatic URL routing without having to make changes to urls.py.